services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java

```diff
@@ -46,7 +46,8 @@
  */
 public class OIDCWellKnownProvider implements WellKnownProvider {
 
-    public static final List<String> DEFAULT_REQUEST_OBJECT_SIGNING_ALG_VALUES_SUPPORTED  = list(Algorithm.none.toString(), Algorithm.RS256.toString());
+    // KEYCLOAK-8460 Request Object Signature Verification Other Than RS256
+    public static final List<String> DEFAULT_REQUEST_OBJECT_SIGNING_ALG_VALUES_SUPPORTED  = list(Algorithm.none.toString(), Algorithm.RS256.toString(), Algorithm.RS384.toString(), Algorithm.RS512.toString(), Algorithm.ES256.toString(), Algorithm.ES384.toString(), Algorithm.ES512.toString());
```

Comment:
See above. Algorithms should not be hardcoded here.

Reply:
I'll do that same way as in Token Signature SPI.